docs/guides/experimental-exporter.md

```diff
@@ -137,6 +137,7 @@ Services are just logical groups of resources used for filtering and organizatio
 * `uc-grants` -  [databricks_grants](../resources/grants.md). *Please note that during export the list of grants is expanded to include the identity that does the export! This is done to allow to creation of objects in case when catalogs/schemas have different owners than the current identity.*.
 * `uc-metastores` - **listing** [databricks_metastore](../resources/metastore.md) and [databricks_metastore_assignment](../resource/metastore_assignment.md) (only on account-level).  *Please note that when using workspace-level configuration, only the metastores from the workspace's region are listed!*
 * `uc-models` - **listing** (*we can't list directly, only via dependencies to top-level object*) [databricks_registered_model](../resources/registered_model.md)
+* `uc-online-tables` - **listing** (*we can't list directly, only via dependencies to top-level object*) [databricks_online_table](../resources/online_table.md)
```

Comment:
Might be good to add that we only support VECTOR_INDEX_FORMAT, MYSQL_FORMAT data source formats for now.

Reply:
They are different - the `MYSQL_FORMAT` is for online tables, and `VECTOR_INDEX_FORMAT` is for vector search (will be separate PR).